apps/hobbies-helsinki/src/domain/search/eventSearch/utils.tsx

```diff
@@ -268,7 +230,7 @@ export const getEventSearchVariables = ({
     isFree: isFree || undefined,
     // internetBased: onlyRemoteEvents || undefined,
     keywordOrSet2: [...(keyword ?? []), ...mappedCategories],
-    keywordOrSet3: [...(keyword ?? []), ...mappedHobbyTypes],
+    keywordOrSet3: [...(keyword ?? [])],
```

Comment:
```
   keywordOrSet2: [...(keyword ?? []), ...mappedCategories],
   keywordOrSet3: [...(keyword ?? [])],
```

If the keyword is already included in the keywordOrSet2, it should not make any difference to list it in a another keywordOrSet. It would just probably make the query a bit slower but keep the result set the same.

Reply:
It's better to remove it completely.

Reply:
Removed in [force push](https://github.com/City-of-Helsinki/events-helsinki-monorepo/compare/0c4b93bf7a97cfcf9931e26e574277fea7a8bcac..c3d7751230097612c20b6a981ca08511b1826dd9).